### PR TITLE
Added comment about supported streaming sources.

### PR DESCRIPTION
**Pull request by aktur**
As suggested.

### CODE REVIEW

**Comment by razzeee** [comment]
Thanks!

Can you change it to a generic third party streaming addons. I'm not sure if we should give a callout to any addon. As it may also be possible that they start supporting this addon, then our readme is wrong ;)

Also it's supposed to work on local files and strm files scrapped to your library.

**Comment by aktur** [comment]
Changed as you suggested.

File: .gitignore [hunk 1]
```diff
@@ -6,3 +6,4 @@
 /.project
 /.pydevproject
 /.settings
+*~
```
Comment (razzeee) [inline .gitignore:9]: What files is this ignoring? Can you give examples?

Comment (aktur) [inline .gitignore:9]: This is for ignoring emacs backup files. Emacs is my IDE of choice. When a file is modified then it puts it old version with extension ~, like README.md~

**Comment by razzeee** [comment]
Thank you!

**Pull request merged**